### PR TITLE
add multiple instance support

### PR DESCRIPTION
**Pull request by bnothomb**

### CODE REVIEW

File: inginious-problems-blockly/static/task_blockly.js [hunk 1]
```diff
@@ -26,4 +26,14 @@ function load_input_blockly(submissionid, key, input) {
...
     }
 
-}
\ No newline at end of file
+}
+
+function load_feedback_blockly(key, content){
+
+    if (typeof Maze !== "undefined" && typeof Maze.reset !== "undefined" && content[0] !== "success") {    
+        var test = $('<div>');
```
Comment (anthonygego) [inline inginious-problems-blockly/static/task_blockly.js]: We should mention in comments here that this is a temporary workaround ;-)

Comment (bnothomb) [reply inginious-problems-blockly/static/task_blockly.js]: I done it ;)

Comment (bnothomb) [reply inginious-problems-blockly/static/task_blockly.js]: Just! I done it now ;)

**Comment by anthonygego** [comment]
OK, and...what do you mean by _suppliple_ ?

**Comment by bnothomb** [comment]
It s mispelling ^^ the correct word was "multiple"

**Pull request merged**